test/units/parsing/utils/test_jsonify.py

```diff
@@ -27,9 +27,7 @@ def test_jsonify_simple(self):
         self.assertEqual(jsonify(dict(a=1, b=2, c=3)), '{"a": 1, "b": 2, "c": 3}')
 
     def test_jsonify_simple_format(self):
-        res = jsonify(dict(a=1, b=2, c=3), format=True)
-        cleaned = "".join([x.strip() for x in res.splitlines()])
-        self.assertEqual(cleaned, '{"a": 1,"b": 2,"c": 3}')
+        self.assertEqual(jsonify(dict(a=1, b=2, c=3), format=True), '{\n    "a":1,\n    "b":2,\n    "c":3\n}')
```

Comment:
yes, both are valid with my PR, as my commit message says, this just simplifies as I don't see the advantage of making the code more complicated.